types/types.go

```diff
@@ -294,6 +306,7 @@ type ContainerJSONBase struct {
 	HostnamePath    string
 	HostsPath       string
 	LogPath         string
+	Node            *ContainerNode
```

Comment:
This should have the `json:"Node,omitempty"` tag, otherwise it will modify the engine's response to add an empty node.